README.md

```diff
@@ -3308,7 +3308,7 @@ Other Style Guides
     ```
 
   <a name="naming--PascalCase-singleton"></a><a name="22.8"></a>
-  - [23.8](#naming--PascalCase-singleton) Use PascalCase when you export a constructor / class / singleton / function library / bare object.
+  - [23.8](#naming--PascalCase-singleton) Use PascalCase when you export-default a constructor / class / singleton / function library / bare object.
```

Comment:
This is an incorrect change; PascalCase *always* applies to these cases, whether named or default exported.

Reply:
If a file has multiple exports, one of which is one of these cases (e.g., class), then that file's name should still match that single export? What if you export multiple classes?

Reply:
Pardon me! I thought this rule was referring to naming the file when you export-default a class. The two rules immediately before refer to naming the file in such cases, and the example uses `export-default`. My mistake!

![image](https://user-images.githubusercontent.com/170023/57053302-f1634680-6c41-11e9-9bf7-24334636c094.png)